### DISCUSSION TITLE
Shelly Bluetooth gateways upported?

### DISCUSSION THREAD

**Discussion by BigHomie90** [custom-components/ble_monitor#1107] [General]
I have a physical adapter using passthrough through virtual box and it was working till now..I am unbale to passthrough the adapter, it fails. Can i use a shelly gen 2 as gateway?

**Comment by Ernst79** [comment]
No, you can’t. BLE monitor doesn’t work with Bluetooth proxies, which can run on Shelly. 

bLE monitor BLE monitor only works with ESPhome BLE gateway, which has the same functionality, but I don’t think it can run on a Shelly https://community.home-assistant.io/t/esphome-ble-gateway-and-other-ble-components/367935

**Reply by BigHomie90** [reply]
Alright then, i ll flash an esp32 with esphome and use that. Thank you.
